typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -340,11 +319,14 @@ class ContextFunder {
       },
     );
 
-    logger.info('Successfully read keys for context from file', {
-      filePath,
-      readOnlyKeysPerChain,
-      context,
-    });
+    logger.info(
```

Comment:
Has the interface for these methods changed?

Reply:
I think it changed a while back but we haven't updated these - none of the metadata here was being logged originally